src/tim/vx/graph.cc

```diff
@@ -195,6 +195,18 @@ void GraphImpl::UpdateTensorConsumersMap(const std::shared_ptr<Tensor>& tensor,
   }
 }
 
+void GraphImpl::RenewTensorConsumersMap(
+    const std::shared_ptr<Tensor>& org_tensor,
+    const std::shared_ptr<Tensor>& dst_tensor, const Operation* op) {
+  for (const auto& added_op : op_vector_) {
```

Comment:
auto exist_op = find( op_vector_.begin(), op_vector_end(), []() { /*lambda*/} );
if (op_vector_.end() == exist_op) { return ; // given op can not be found }

auto consumer_to_remove = tensor_consumers_.find(org_tensor);
if ( tensor_consumer_.end() != consumer_to_remove ) tensor_consumers_.erase(...);

tensor_conumsers_[dst_tensor].push_back(added_op);

Reply:
Modified and checked ut success